kevlar/assemble.py

```diff
@@ -123,7 +123,7 @@ def assemble_with_greed(graph, ccindex, debugout=None):
         count += 1
 
         pair = fetch_largest_overlapping_pair(graph)
-        newname = 'contig{:d};cc={:d}'.format(count, ccindex)
+        newname = 'contig{:d}:cc={:d}'.format(count, ccindex)
```

Comment:
These contig names end up in the VCF file, where semicolons are used as delimiters for key/value pairs